test/python/circuit/test_circuit_load_from_qpy.py

```diff
@@ -1685,6 +1685,18 @@ def test_qpy_deprecation(self):
             # pylint: disable=no-name-in-module, unused-import, redefined-outer-name, reimported
             from qiskit.circuit.qpy_serialization import dump, load
 
+    @ddt.data(0, "01", [1, 0, 0, 0])
+    def test_valid_circuit_with_initialize_instruction(self, param):
+        """Tests that circuit that has initialize instruction can be saved and correctly retrieved"""
+        qc = QuantumCircuit(2)
+        qc.initialize(param, qc.qubits)
+        with io.BytesIO() as fptr:
+            dump(qc, fptr)
+            fptr.seek(0)
+            new_circuit = load(fptr)[0]
+        self.assertEqual(qc, new_circuit)
+        self.assertDeprecatedBitProperties(qc, new_circuit)
```

Comment:
Yeah, we could potentially improve this.  This particular file is older that `test/qpy/test_circuit_load_from_qpy.py`, and to be honest, I don't know why this one still exists - I thought it had been moved into the `test/python/qpy` one some time ago.  You're right that it would be better in a follow-up PR.  If you'd like to make such a PR, it would be welcome.